utilities/unittests/test_storage.py

```
@@ -107,8 +107,8 @@ def test_registry_source_with_secret_and_cert(self):
             }
         }
 
-    @patch.dict("utilities.storage.py_config", {"cpu_arch": "arm64", "cluster_type": "multiarch"})
-    def test_registry_source_multiarch_with_cpu_arch(self):
+    @patch("utilities.storage.get_multiarch_cpu_arch", return_value="arm64")
+    def test_registry_source_multiarch_with_cpu_arch(self, mock_get_arch):
```

Ruff (0.16.0)

[warning] Unused method argument: `mock_get_arch`

(ARG002)